.gitignore

```diff
@@ -14,3 +14,6 @@ vendor
 
 # PhpUnit configuration local machine only
 phpunit.xml
+
+#IDE env
+.idea
```

Comment:
This should more be in your `.gitignore_global`, as it is specific to your environment